### DISCUSSION TITLE
Best Cloud Architecture for OME

### DISCUSSION THREAD

**Discussion by plawansai** [OvenMediaLabs/OvenMediaEngine#1162] [Help]
Hi all!

We're interesting OME and we are happy with how it is working. So, we decided to buy a new cloud after the tests. My question is which architecture do you recommend running OME.

May I have a hint from you guys in order to decide this?

Thank you very much, OME is working fine for us! :)

**Comment by naanlizard** [comment]
Whatever you find, perhaps it will be relevant in this thread. I've got a similar thing I'm investigating

https://github.com/AirenSoft/OvenMediaEngine/discussions/1153

**Comment by getroot** [comment]
The discussion below may also be helpful to you.

https://github.com/AirenSoft/OvenMediaEngine/discussions/1080
